tsdb/series_file.go

```diff
@@ -89,6 +93,11 @@ func (f *SeriesFile) Close() (err error) {
 	return err
 }
 
+// Close unmaps the data file.
+func (f *SeriesFile) Close() (err error) {
+	return f.doClose(true)
```

Comment:
Instead of doing this, you can just do this:

```go
func (f *SeriesFile) Close() (err error) {
    f.refs.Lock()
    defer f.refs.Unlock()
    return f.close()
}
```

Reply:
changed as suggested